packages/block-renderer/src/hooks/use-rendered-patterns.ts

```diff
@@ -56,7 +60,7 @@ const useRenderedPatterns = (
 				pages.reduce( ( previous, current ) => ( { ...previous, ...current } ), {} )
 			);
 		} );
-	}, [] );
+	}, [ patternIds ] );
```

Comment:
It might be better to use `patternIds.length` to avoid requesting the endpoint after each re-render. Or we can try to memo [the encoded pattern ids](https://github.com/Automattic/wp-calypso/blob/56a69d7471be79536ccc3cf5e35fa15ca18adffa/client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx#L612) to avoid creating the new one on every re-render.

Reply:
Nice catch, done.